src/Controllers/NIPSWeb/confirm_central_upload.php

```diff
@@ -11,7 +11,7 @@
     $_REQUEST['artist'],
     $_REQUEST['album'],
     $_REQUEST['position'],
-    $_REQUEST['explicit'] ? true : null
+    filter_var($_REQUEST['explicit'], FILTER_VALIDATE_BOOLEAN) ? true : null
```

Comment:
From the docs for filter_var: "Returns the filtered data, or FALSE if the filter fails."

Seems to me that this won't be able to differentiate between `false` and some not-boolean value. Sure this is the right thing to do?